tools/accuracy_checker/configs/rfcn_resnet101_coco_tf.yml

```
@@ -0,0 +1,48 @@
+models:
+  - name:  rfcn_resnet101_coco_tf
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/rfcn_resnet101_coco_tf/FP32/rfcn_resnet101_coco_tf.xml
```

score used in directory and files name in models
```suggestion
        model:   public/rfcn-resnet101-coco-tf/FP32/rfcn-resnet101-coco-tf.xml
```

---

please update all paths in AC config, file name and model name respectively